src/libs/actions/Task.js

```diff
@@ -424,21 +424,24 @@ function setShareDestinationValue(shareDestination) {
  * It also sets the shareDestination as that chat report if a share destination isn't already set
  * @param {string} assignee
  * @param {string} shareDestination
+ * @param {boolean} isCurrentUser
  */
 
-function setAssigneeValue(assignee, shareDestination) {
-    let newChat = {};
-    const chat = ReportUtils.getChatByParticipants([assignee]);
-    if (!chat) {
-        newChat = ReportUtils.buildOptimisticChatReport([assignee]);
-    }
-    const reportID = chat ? chat.reportID : newChat.reportID;
+function setAssigneeValue(assignee, shareDestination, isCurrentUser = false) {
+    if (!isCurrentUser) {
+        let newChat = {};
+        const chat = ReportUtils.getChatByParticipants([assignee]);
+        if (!chat) {
+            newChat = ReportUtils.buildOptimisticChatReport([assignee]);
+        }
+        const reportID = chat ? chat.reportID : ReportUtils.buildOptimisticChatReport([assignee]).reportID;
```

Comment:
```suggestion
        const reportID = chat ? chat.reportID : newChat.reportID;
```

Apologies, nevermind, we still need the `newChat` so there's no point in this